framework/caching/FileDependency.php

```diff
@@ -45,7 +45,11 @@ protected function generateDependencyData($cache)
         }
 
         $fileName = Yii::getAlias($this->fileName);
+        if (!file_exists($fileName)) {
+            return 0;
+        }
+
         clearstatcache(false, $fileName);
-        return @filemtime($fileName);
+        return filemtime($fileName);
```

Comment:
This change is incorrect - now this operation is not atomic and it may throw errors on high concurrency workloads.

Reply:
I made a reversal of these changes